src/classes/RD2_ElevateInformation_CTRL.cls

```diff
@@ -101,6 +101,8 @@ public with sharing class RD2_ElevateInformation_CTRL {
                 data.put('commitmentURLPrefix', service.getViewURLPrefix(PS_IntegrationServiceConfig.VIEW_TYPE.COMMITMENT));
             }
 
+            data.put('closedStatus', RD2_StatusMapper.getInstance().getClosedStatusValues());
```

Comment:
Could this key be `closedStatuses` instead since there can be more than one status defined as closed?